backend/app/mcp/server.py

```
@@ -0,0 +1,342 @@
+
+    # Set up task context
+    with TaskContextManager(
+        task_id=request.task_id, subtask_id=request.subtask_id
+    ):
+        try:
+            if request.tool_name == "send_message":
+                from app.mcp.tools.send_message import send_message
+
+                # Parse attachments if provided
+                attachments = None
+                if "attachments" in request.arguments and request.arguments["attachments"]:
```

Ruff (0.14.13)

Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)